core/src/main/java/net/miginfocom/layout/Grid.java

```diff
@@ -217,8 +217,8 @@ public Grid(ContainerWrapper container, LC lc, AC rowConstr, AC colConstr, Map<?
 			// Move to a free y, x  if no absolute grid specified
 			int cx = rootCc.getCellX();
 			int cy = rootCc.getCellY();
-			if ((cx < 0 || cy < 0) && rowNoGrid == false && rootCc.getSkip() == 0) { // 3.7.2: If skip, don't find an empty cell first.

```

Comment:
Just a pondering: from a readability aspect, there is something to be said for "== false" instead of "!"... My mind at least parses "rowNoGrid is false" much easier than "not rowNoGrid". Different are "is" situations; like "isEmpty" vs "!isEmpty", there the condition reads naturally. So I would not apply this change indiscriminately

Reply:
@tbee I think, better name for `rowNoGrid` will be to remove `No` to avoid double negation and mental confusion, right?

Reply:
That is another aspect, but I'm referring to the fact that all == false have been replaced with ! in this PR; ! is not necessarily better than == false.

Reply:
`== false` it is religious :( I can say that I never use or see such code style (except JS/TS where we need strict equality `=== false` because of `undefined`). I think that it is not "another aspect", and name like `isRowInGridMode` will reduce your confusion.

Reply:
Correct. So replacing "rowNoGrid == false" with "isRowInGridMode" is perfect, consequently just replacing all "== false" with "!" is not. I'll leave it at this. I just wanted to point out that ! is not necessarily better that == false, just because it is shorter.

Reply:
@tbee Thanks for review, addressed in #57